third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -895,19 +896,16 @@ struct ClampFOpConversion
 
   explicit ClampFOpConversion(LLVMTypeConverter &typeConverter,
                               ModuleAxisInfoAnalysis &axisAnalysisPass,
-                              int computeCapability, PatternBenefit benefit = 1)
+                              int computeCapability,
+                              PatternBenefit benefit = patternBenefitDefault)
       : ElementwiseOpConversionBase(typeConverter, axisAnalysisPass, benefit),
         computeCapability(computeCapability) {}
 
-  SmallVector<Value> createDestOps(ClampFOp op, OpAdaptor adaptor,
-                                   ConversionPatternRewriter &rewriter,
-                                   Type elemTy, MultipleOperandsRange operands,
-                                   Location loc) const {
+  bool isClipPattern(ClampFOp op) const {
```

Comment:
From what Marat said, there is a way of lowering the clip pattern to single op on AMD too. Maybe `isClipPattern` could go to the common file so it can be referenced by both backends?

Reply:
Is it the same pattern though? Also, currently the optimized codegen is using inline ptx, from my discussion with @ThomasRaoux seems like we want to keep the optimized implementation for later.